app/_src/gateway/production/monitoring/health-check.md

```diff
@@ -0,0 +1,90 @@
+---
+title: Health Check
+content_type: tutorial
+---
+
+This tutorial will guide you through the process of using the new Kong Status endpoint, which provides a more reliable and simple way to determine if Kong is ready to serve requests.
```

Comment:
```suggestion
This tutorial will guide you through the process of using the new Node Readiness Endpoint, which provides a more reliable and simple way to determine if Kong is ready to serve user requests.
```

Some consistencies in what this should be called is probably a good thing. Suggest we use "Node Readiness Endpoint" from now on.

Specifically, "Status endpoint" is a confusing name because we have a couple of "status endpoints", including `/status`.